src/components/dashboard/users/AccessContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import globalize from '../../../scripts/globalize';
+import CheckBoxElement from './CheckBoxElement';
+
+type IProps = {
+    ContainerClassName?: string;
```

Comment:
I think we should probably standardize on camelCase for prop names.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3076/commits/0a881d6f6f1c87efb1a5330499b676d9991461c2